### ISSUE TITLE
Improve functionality

### ISSUE THREAD

**Issue by grzegorzsurowiec** [thundernest/import-export-tools-ng#480]
Hello everyone.
This is not a bug but more of a request to improve functionality.
When exporting a folder to EML format with an index, the subfolder option is missing.
I have over 100 folders to export and I'm doing it manually for now, it would be nice to be able to export files from all subfolders as well.

**Comment by cleidigh** [comment]
@grzegorzsurowiec 
Yes indeed, this is an important missing functionality. Basically structured, subfolder export is needed for all the folder message types

This is basically the the top enhancement I need to do, I just have so much to do....
When I get v14 settled I begin doing I improvements.

@cleidigh

**Comment by cleidigh** [comment]
@grzegorzsurowiec 

Finally got the first pass on this!
Track and help beta here:
- #538 

@cleidigh

**Comment by rste79** [comment]
Hi!
I see that work for exporting recursively is already done but it has focused only on EML, HTML, PDF, Plaintext format with full messages and optionally indexes.
Is it possible to add recursive export also to the other export option, i.e. without messages but only indexes ("Message Index" -> "CSV Format" and "HTML Format") ?
